app/game.rb

```diff
@@ -22,7 +24,7 @@ def update
     @bullets.update
     if @map.colliding? @player then
       @map.stop_moving!
-      @player.die!
+      @player.remove_life
```

Comment:
nit: rename the method to remove_life! because it changes the state of the object.